Header header
int32 count
RecognizedObject[] objects
